java/ql/lib/semmle/code/java/security/Encryption.qll

```diff
@@ -250,7 +250,8 @@ string getASecureAlgorithmName() {
   result =
     [
       "RSA", "SHA-?256", "SHA-?512", "CCM", "GCM", "AES(?![^a-zA-Z](ECB|CBC/PKCS[57]Padding))",
-      "Blowfish", "ECIES"
+      "Blowfish", "ECIES", "RSA/ECB/OAEPWithSHA-1AndMGF1Padding",
+      "RSA/ECB/OAEPWithSHA-256AndMGF1Padding"
```

Comment:
This should no longer be necessary, since `RSA/ECB/*` is covered by the `RSA` string at the beginning of the list, and in the context where `getASecureAlgorithmName` is used there's no interaction with `getAnInsecureAlgorithmName` that could cause a conflict.

```suggestion
      "Blowfish", "ECIES"
```